src/themes/appletv/theme.css

```diff
@@ -293,7 +293,20 @@ html {
 
 .emby-checkbox:checked + span + .checkboxOutline,
 .itemProgressBarForeground {
-    background-color: #00a4dc;
+    background: rgb(0, 210, 201);
+    background: -moz-linear-gradient(90deg, rgba(0, 210, 201, 1) 0%, rgba(13, 194, 98, 1) 28%, rgba(0, 75, 185, 1) 100%);
+    background: -webkit-linear-gradient(90deg, rgba(0, 210, 201, 1) 0%, rgba(13, 194, 98, 1) 28%, rgba(0, 75, 185, 1) 100%);
```

Comment:
No need for fallbacks or vendor prefixes.

The browsers we target (Visible in `package.json` under the Browserslist key) all support CSS gradient (See https://caniuse.com/#feat=css-gradients) and we have a tool in the build process that handles vendor prefixing automatically (We've actually been meaning to remove all the prefixes we currently have).

Reply:
Thanks, I will change